src/core/udb-api.service.js

```diff
@@ -231,6 +231,41 @@ function UdbApi(
     return deferredOffer.promise;
   };
 
+  /**
+   * @param {Array} events
+   * @return {Array}
+   */
+  this.reformatEvents = function(events) {
```

Comment:
This does not only reformat `events` but also `places` and `organizers` (json data). Could we rename it to something like `reformatJsonLDData` instead? The reformatting it does is very related to generic JSON-LD data  (`@context` and `@type` properties).